pkgs/by-name/du/durden/package.nix

```diff
@@ -27,6 +28,8 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru.updateScript = nix-update-script {};
```

Comment:
Isn't a more specialized function better here?

Reply:
Maybe. It works fine this way. I used it to update the version.

update.nix complains if it's not defined

Reply:
Ok then.